verdaccio.yml

```diff
@@ -0,0 +1,31 @@
+storage: ./storage
+auth:
+  htpasswd:
+    file: ./htpasswd
+uplinks:
```

Comment:
Do we really need to define the uplinks?

Reply:
I believe we do, otherwise we run into packages not being found